products/aws-lambda.md

```diff
@@ -0,0 +1,304 @@
+---
+title: AWS Lambda
+category: service
+tags: amazon
+iconSlug: awslambda
+permalink: /aws-lambda
+releasePolicyLink: https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html
+releaseDateColumn: true
+releaseColumn: false
+activeSupportColumn: Deprecation Phase 1
+eolColumn: Deprecation Phase 2
```

Comment:
```suggestion
activeSupportColumn: Standard Support
eolColumn: Deprecated Support
```

Dates on the upstream website are *starting dates* for each phase, while we document ending-dates. See this image for context:

![image](https://github.com/endoflife-date/endoflife.date/assets/584253/f4170e9a-ebc8-477d-afb7-359dbd1bc5b3)

`activeSupportColumn` in this case is supposed to document the end of the green period, so we should title it accordingly (Standard Support / "Support"). The wording in the table would accordingly match as well (Support ends ...)

`eolColumn` is to document the end of the orange period, so we should just call it End-of-Life or Deprecated Support.

For a concrete example: 

- `2023-12-07` is when Ruby2.7 Deprecation Phase 1 begins, and Standard Support ends.
- `2024-01-07` is when Ruby2.7 Deprecation Phase 2 begins, and Deprecation Phase 1 ends.

Makes sense to document it as the latter.

Reply:
You are right, `Deprecation Phase x` are not good names. Will use what you suggested.